xml/deploy_cephadm.xml

```diff
@@ -622,6 +624,16 @@ o- time_server ................................................ [enabled]
       </para>
      </listitem>
     </itemizedlist>
+    <tip>
+     <title>Registry Cache</title>
+     <para>
+      To avoid re-syncing the local registry when new updated containers
+      appear, you can configure a <emphasis>registry cache</emphasis>. Find
+      more information about containers and air-gapped scenarios in
+      <link
+       xlink:href="https://documentation.suse.com/suse-caasp/4.1/html/caasp-deployment/_deployment_scenarios.html#_airgapped_deployment"/>.
```

Comment:
Maybe add a `TODO` that this needs to be updated when CaaSP 5 got released?

Reply:
see https://github.com/SUSE/doc-ses/issues/389